projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -22,15 +22,20 @@ export class NgSelectA11yDirective implements AfterViewInit {
     const divCombobox =
       this.elementRef.nativeElement.querySelector('[role="combobox"]');
 
-    this.renderer.setAttribute(
-      divCombobox,
-      'aria-label',
-      this.cxNgSelectA11y.ariaLabel
-    );
-    this.renderer.setAttribute(
-      divCombobox,
-      'aria-controls',
-      this.cxNgSelectA11y.ariaControls
-    );
+    if (this.cxNgSelectA11y.ariaLabel) {
```

Comment:
why check it if it's a required field?

Reply:
Well the object literal may be passed as an input but it is possible to not pass the property ariaLabel inside the object

Reply:
@Pucek9 so the field is not strictly required